coordinator/src/app.ts

```diff
@@ -2,11 +2,12 @@ import {Boom} from "@hapi/boom"
 import Hapi from "@hapi/hapi"
 import routes from "./routes"
 import {toOperationOutcome, FhirMessageProcessingError} from "./models/errors/processing-errors"
+import HapiPino from "hapi-pino"
 
 const preResponse = function (request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) {
   const response = request.response
   if (response instanceof Boom) {
-    console.log(response)
+    request.log("info", response)
     if (response instanceof FhirMessageProcessingError) {
```

Comment:
Might be nice to log FhirMessageProcessingError at info level and everything else at error level?

Reply:
Done :+1: